src/briefcase/platforms/linux/appimage.py

```diff
@@ -154,7 +160,11 @@ def output_format_template_context(self, app: AppConfig):
         # Add the manylinux tag to the template context.
         try:
             tag = getattr(app, "manylinux_image_tag", "latest")
-            context["manylinux_image"] = f"{app.manylinux}_{self.tools.host_arch}:{tag}"
+            manylinux_arch = {
+                "x86_64": "x86_64",
+                "i386": "i686",
```

Comment:
linuxdeploy isn't available for `aarch64` yet. So, I figured without a way to actually test such a platform, it should wait until support for linuxdeploy on arm64 is added.

Reply:
Oh - for sure - but then, building for x86_64 doesn't work either, because the docker container doesn't match the local CPU architecture. `docker run` has to be started with an explicit `--platform=linux/amd64` if you want to run x86 images on M1.

Reply:
Yeah....but it _should_ work :) (details in other comment: https://github.com/beeware/briefcase/pull/1360#discussion_r1278785496)

If this can't be resolved (and it's looking that way), I think it may be best to disable AppImage builds when not using x86 hardware.

Reply:
> `docker run` has to be started with an explicit `--platform=linux/amd64` if you want to run x86 images on M1.

Just to be clear, this isn't explicitly true. If you try to run an image that isn't multi-platform, Docker will just try to run the image regardless....and if it works, it works. Therefore, given the manylinux images are not multi-platform, Docker will just try to run whichever one you specify....however, you will get warning about the mismatch of architectures.

Reply:
> Yeah....but it _should_ work :) (details in other comment: [#1360 (comment)](https://github.com/beeware/briefcase/pull/1360#discussion_r1278785496))
> 
> If this can't be resolved (and it's looking that way), I think it may be best to disable AppImage builds when not using x86 hardware.

Looking at this more, I definitely don't see a simple or straightforward workaround. The x86-64 QEMU emulation provided by Docker Desktop on Apple Silicon just isn't currently compatible with AppImages.

Based on this, I'll just disable building AppImages on arm64 macOS.

Refs:
- https://github.com/AppImage/AppImageKit/issues/965#issuecomment-1333557171
- https://github.com/AppImage/AppImageKit/issues/1056
- Possible upstream QEMU root issue: https://gitlab.com/qemu-project/qemu/-/issues/927

Reply:
Disabling is sounding like the safest approach. When combined with the fact that we've de-emphasized AppImage builds in favor of system builds, going to extraordinary efforts to debug why linuxdeploy won't work on x86 emulation sounds like effort that would be better spend elsewhere.